webapp/components/posts_view.jsx

```diff
@@ -550,18 +542,6 @@ export default class PostsView extends React.Component {
                         </div>
                     </div>
                 </div>
-                <div
-                    className='post-list__new-messages-below'
-                    onClick={this.scrollToBottomAnimated}
-                    hidden={!this.state.showUnreadMessageAlert}
-                >
-                    <i className='fa fa-angle-down'></i>
-                    &nbsp;
-                    <FormattedMessage
-                        id='posts_view.newMsg'
```

Comment:
Never mind, it's still used further up for the old New Messages indicator https://github.com/hmhealey/mattermost-platform/blob/plt2757/webapp/components/posts_view.jsx#L313